package.json

```diff
@@ -84,6 +84,10 @@
 		"test:blackbox:geth:ws": "yarn pre-blackbox && yarn geth:start:background && ./scripts/verdaccio.sh startBackgroundAndPublish && lerna run test:blackbox:geth:ws --stream && yarn post-blackbox:geth",
 		"test:blackbox:infura:http": "yarn pre-blackbox && ./scripts/verdaccio.sh startBackgroundAndPublish && lerna run test:blackbox:infura:http --stream && yarn post-blackbox",
 		"test:blackbox:infura:ws": "yarn pre-blackbox && ./scripts/verdaccio.sh startBackgroundAndPublish && lerna run test:blackbox:infura:ws --stream && yarn post-blackbox",
+		"test:stress:data": "packages/web3/test/stress/large_request_data/start.sh",
+		"test:stress:requests": "packages/web3/test/stress/many_requests/start.sh",
+		"test:stress:validation": "node packages/web3/test/stress/large_validator_data/index.js",
+		"test:stress": "yarn test:stress:data && yarn test:stress:validation && yarn test:stress:requests",
```

Comment:
What do you think about prefixing this with `test:manual:stress` instead of `test:stress`? So, we use this convention of `test:manual:` for tests that are supposed to run manually. Or, do you think this could be run also in CI sometime?